react/Icons/Info.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FunctionComponent } from 'react'
+
+const InfoIcon: FunctionComponent = () => {
+  return (
+    <svg
+      width="16"
+      height="16"
```

Comment:
Add className in the format:

`vtex__icon-NAME`

Following styleguide's icons:
https://github.com/vtex/styleguide/blob/master/react/components/icon/Help/index.js

https://github.com/vtex/styleguide/blob/master/react/components/icon/utils.js#L16

Reply:
Added this to both icons